recipe/meta.yaml

```diff
@@ -10,37 +10,56 @@ source:
   sha256: 0e928e9a07566b7b8d781169f5fa1bcd9e7ae196317d4a433b692802bed499e6
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
+  missing_dso_whitelist:  
+    - /lib64/libpthread.so.0
+    - /lib64/libc.so.6
+    - /lib64/libm.so.6
+    - /lib64/ld-linux-x86-64.so.2
```

Comment:
Can you elaborate on the issue that required `missing_dso_whitelist` to be specified, and how this solves the problem?

Reply:
Example: [failing CI](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=801339&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=22028). 

Similarly to case in other comment, using `outputs` section causes stricter checks. Error is due to `sqlalchemy `, `pydantic` and some other packages are linked to these standard linux libraries but these libraries are not explicitly listed in the recipe dependencies. There is no reason for us to ship standard linux libraries that we don't have direct dependencies on. Adding them to `missing_dso_whitelist `would tell conda build to ignore their linkage information during checking ([docs section](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#whitelisting-shared-libraries)).